### PR TITLE
Feat/manager 14180 : Credential step 3

### PR DESCRIPTION
**Pull request by vovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/key-management-service-B3
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-14180
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

**Review by anooparveti — CHANGES_REQUESTED** [review]
The components added in the `components` folder are limited to Create Credential. So, add them in the `packages/manager/apps/key-management-service/src/pages/credential/create` folder itself.

**Review by tibs245 — APPROVED** [review]

File: packages/manager/apps/key-management-service/src/components/credential/create/confirmation/CreateCredentialConfirmation.component.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  okmsCredential,
+}: CreateCredentialConfirmationProps) => {
+  const { t } = useTranslation('key-management-service/credential');
+  const [isKeyDownloaded, setIsKeyDownloaded] = useState(false);
+
+  return (
+    <div className="grid grid-cols-1 gap-4 md:gap-6 max-w-screen-lg">
+      <CreateCredentialConfirmationDetails okmsCredential={okmsCredential} />
+      <CreateCredentialConfirmationPrivateKey
+        privateKey={okmsCredential.certificatePEM}
```
Comment (rjamet-ovh) [inline packages/manager/apps/key-management-service/src/components/credential/create/confirmation/CreateCredentialConfirmation.component.tsx:23]:
```suggestion
        privateKey={okmsCredential.privateKeyPEM}
```

File: packages/manager/apps/key-management-service/src/components/credential/create/confirmation/CreateCredentialConfirmationPrivateKey.component.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            'key_management_service_credential_create_confirmation_private-key_warn',
+          )}
+        </OsdsMessage>
+        <span>
+          <OsdsButton
+            variant={ODS_BUTTON_VARIANT.stroked}
+            color={ODS_THEME_COLOR_INTENT.primary}
+            href={`data:text/json;charset=utf-8,${encodeURIComponent(
+              JSON.stringify(privateKey),
+            )}`}
```
Comment (rjamet-ovh) [inline packages/manager/apps/key-management-service/src/components/credential/create/confirmation/CreateCredentialConfirmationPrivateKey.component.tsx:55]:
```suggestion
            href={`data:application/x-pem-file;base64,${btoa(privateKey)}`}
```

File: packages/manager/apps/key-management-service/src/components/credential/create/confirmation/CreateCredentialConfirmation.component.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        credentialId={okmsCredential.id}
+        isKeyDownloaded={isKeyDownloaded}
+        setIsKeyDownloaded={setIsKeyDownloaded}
+      />
+      <span>
+        <OsdsButton
+          color={ODS_THEME_COLOR_INTENT.primary}
+          disabled={!isKeyDownloaded || undefined}
+          inline
```
Comment (rjamet-ovh) [inline packages/manager/apps/key-management-service/src/components/credential/create/confirmation/CreateCredentialConfirmation.component.tsx:32]: `onClick` is missing

**Review by anooparveti — APPROVED** [review]

**Review by rjamet-ovh — APPROVED** [review]

**Pull request merged**